### PR TITLE
Hw5 oleksii fedorchuk

### PR DESCRIPTION
**Pull request by Oleksii-Fedorchuk**

### CODE REVIEW

File: Oleksii_Fedorchuk/HW5_Oleksii_Fedorchuk/task2.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+friends = ["John", "Marta", "James", "Amanda", "Marianna"]
+title_string = ' Names '
+print(f"{title_string}".center(50, "*"))
+for names in friends:
+    print(f"{friends[0]}".rjust(27))
+    print(f"{friends[1]}".rjust(27))
+    print(f"{friends[2]}".rjust(27))
+    print(f"{friends[3]}".rjust(27))
+    print(f"{friends[4]}".rjust(27))
+    break
```
Comment (bohdandovbsyh) [inline Oleksii_Fedorchuk/HW5_Oleksii_Fedorchuk/task2.py]: Em. Very strange solution. Why do you use a loop if you get value by indexes inside it?

File: Oleksii_Fedorchuk/HW5_Oleksii_Fedorchuk/task5.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+           "but it is one of the largest and most versatile, renowned both as a vital research tool and as a public " \
+           "relations boon for astronomy. The Hubble telescope is named after astronomer Edwin Hubble and is one of " \
+           "NASA's Great Observatories..... The Space Telescope Science Institute (STScI) selects Hubble's targets " \
+           "and processes the resulting data, while the Goddard Space Flight Center (GSFC) controls the spacecraft.A " \
+           "commission headed by Lew Allen, director of the Jet Propulsion Laboratory, was established to determine " \
+           "how the error could have arisen. The Allen Commission found that a reflective null corrector, " \
+           "a testing device used to achieve a properly shaped non-spherical mirror, had been incorrectly " \
+           "assembled—one lens was out of position by 1.3 mm (0.051 in). "
+    for item in re.split(r"[.?!]\s", text):
+        print(item)
```
Comment (bohdandovbsyh) [inline Oleksii_Fedorchuk/HW5_Oleksii_Fedorchuk/task5.py:16]: Does not work correctly. Please check your solution

File: Oleksii_Fedorchuk/HW5_Oleksii_Fedorchuk/task2.py [hunk 3]
```diff
@@ -1,10 +1,19 @@
...
-    print(f"{friends[2]}".rjust(27))
-    print(f"{friends[3]}".rjust(27))
-    print(f"{friends[4]}".rjust(27))
-    break
+print(f"{friends[0].rjust(27)}")
+print(f"{friends[1].rjust(27)}")
+print(f"{friends[2].rjust(27)}")
+print(f"{friends[3].rjust(27)}")
+print(f"{friends[4].rjust(27)}")
+
```
Comment (bohdandovbsyh) [inline Oleksii_Fedorchuk/HW5_Oleksii_Fedorchuk/task2.py]: Why  do not you  just use:
for friend in friends:
       print(friend.rjust(27))

Comment (Oleksii-Fedorchuk) [reply Oleksii_Fedorchuk/HW5_Oleksii_Fedorchuk/task2.py]: Yes, you are right. It's  much easier.

File: Oleksii_Fedorchuk/HW5_Oleksii_Fedorchuk/task5.py (hunk 2 above)
Comment (bohdandovbsyh) [reply Oleksii_Fedorchuk/HW5_Oleksii_Fedorchuk/task5.py:16]: The Hubble.Space.Telescope - is not a sentence. 
The problem is in:
The Space Telescope Science Institute (STScI) selects Hubble's targets and processes the resulting data, while the Goddard Space Flight Center (GSFC) controls the spacecraft.A commission headed by Lew Allen, director of the Jet Propulsion Laboratory, was established to determine how the error could have arisen

It is 2 sentences, but your code do not split them, and print as 1

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**